src/System.Security.Cryptography.Algorithms/src/Resources/Strings.resx

```diff
@@ -176,7 +176,7 @@
     <value>This operation is not supported for this class.</value>
   </data>
   <data name="Cryptography_InvalidPadding" xml:space="preserve">
-    <value>Specified padding mode is not valid for this algorithm.</value>
+    <value>Padding is invalid and cannot be removed.</value>
```

Comment:
Is this as useful as we can make it? Would the API caller know what to do here?

Reply:
> Would the API caller know what to do here?

According to StackOverflow? No :smile:.  It means any of

* You decrypted with the wrong key
* You have a one block CBC message and provided the wrong IV
* You are using something from the future which carries the IV further in the decryption pipeline than CBC and you provided the wrong IV
* You are decrypting in a padding mode other than the message was encrypted in
* You are decrypting something which isn't an encrypted message (similar to, but technically different than "wrong key")
* You called TransformFinalBlock but you aren't actually on the final block
* You provided incomplete data to TransformFinalBlock

Similar to a compiler emitting the good old `expected ';'` message it's a problem of at this low of a level we don't have any idea what you did to get here, we just know that we went "click" and it went "hum" and that's not the right answer.

> Is this as useful as we can make it?

Pretty much. ::initiates robots-falling-out-of-the-hole-in-the-ship sequence::